plugins/destination/kafka/client/transformer.go

```diff
@@ -1,190 +1,73 @@
 package client
 
-import "github.com/cloudquery/plugin-sdk/schema"
+import (
+	"github.com/cloudquery/plugin-sdk/schema"
+)
 
 func (c *Client) TransformBool(v *schema.Bool) any {
-	switch c.pluginSpec.Format {
-	case FormatTypeCSV:
-		return c.csvTransformer.TransformBool(v)
-	case FormatTypeJSON:
-		return c.jsonTransformer.TransformBool(v)
-	default:
-		panic("unknown format " + c.pluginSpec.Format)
-	}
+	return c.filetype.TransformBool(v)
```

Comment:
embed & drop transformers

Reply:
Can't embed because there is no `BatchWrite` like the others

Reply:
why is this a problem?